create_tarball.sh

```diff
@@ -12,10 +12,22 @@ TARBALL="tripleo-ui-deps-$REVIEW.$PATCHSET.tar.gz"
 function generate_archive() {
     rm -rf $REPONAME
     git clone $REPO
+    cat <<-EOF > Dockerfile
+    FROM centos:centos7
+    RUN curl -o /etc/yum.repos.d/deps.repo https://trunk.rdoproject.org/centos7-rocky/delorean-deps.repo
+    RUN yum -y install npm
+    VOLUME ["/src"]
+    WORKDIR /src
+    CMD ["npm","install"]
```

Comment:
By default, it is run as root, which is okay-ish as we're running in a container, but we can make it work as a non-privileged user.